.storybook/main.js

```diff
@@ -27,4 +27,19 @@ module.exports = {
     ...config,
     LOGGER: true,
   }),
+  typescript: {
```

Comment:
Options to autogenerate types from typescript
It works for direct props of the component and not nested types. Thus the usage of argTypes as well to document the nested types